sway/desktop/output.c

```diff
@@ -822,8 +821,7 @@ static void handle_destroy(struct wl_listener *listener, void *data) {
 	update_output_manager_config(server);
 }
 
-static void handle_mode(struct wl_listener *listener, void *data) {
-	struct sway_output *output = wl_container_of(listener, output, mode);
+static void output_mode(struct sway_output *output) {
```

Comment:
Nit: can we keep the original function name?

Reply:
Of course